src/main/java/org/ethereum/vm/client/Repository.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) [2018] [ The Semux Developers ]
+ * Copyright (c) [2016] [ <ether.camp> ]
+ * This file is part of the ethereumJ library.
+ *
+ * The ethereumJ library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The ethereumJ library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with the ethereumJ library. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.ethereum.vm.client;
+
+import java.math.BigInteger;
+
+import org.ethereum.vm.DataWord;
+
+public interface Repository {
+
+    /**
+     * Checks whether an account exists.
+     *
+     * @param address
+     *            the account address
+     * @return true if account exist, false otherwise
+     */
+    boolean isExist(byte[] address);
+
+    /**
+     * Creates an account if not exist.
+     *
+     * @param address
+     *            the account address
+     * @ImplNote trigger account creation
+     */
+    void createAccount(byte[] address);
```

Comment:
Is this createAccount/deleteAccount compatible with our database?
It's not immediately clear if accounts in VM != addresses ?